CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Manage user-funder permissions using `PUT`, and `DELETE` on `/user/{keycloakUserId}/funders/{funderShortCode}/permissions/{permission}`.
 - Manage user-data-provider permissions using `PUT`, and `DELETE` on `/user/{keycloakUserId}/funders/{funderShortCode}/permissions/{permission}`.
+- Manage user-group-changemaker permissions using `PUT`, and `DELETE` on `/userGroups/{keycloakOrganizationId}/changemaker/{changemakerId}/permissions/{permission}`.
+- Manage user-group-data-provider permissions using `PUT`, and `DELETE` on `/userGroups/{keycloakOrganizationId}/dataProvider/{dataProviderShortCode}/permissions/{permission}`
+- Manage user-group-funder permissions using `PUT`, and `DELETE` on `/userGroups/{keycloakOrganizationId}/funders/{funderShortCode}/permissions/{permission}`
```

Comment:
You reminded me that I forgot to update the changelog for some other entities, whoops! Thank you!

My following comment is not particular to this pull request, but more for the team in general. I don't think it makes sense to put stuff under "unreleased" at this stage of the PR, because once it gets merged, it will be released. So I suggest incrementing the patch or more likely minor version (how do we decide, @slifty?) in the OpenAPI spec and guessing at a date here, maybe today. If it turns out you are going to smash "merge" tomorrow, then update the date before doing so.